Pango-1.0.gir

```diff
@@ -1652,9 +1652,9 @@ If you are using Pango as part of a higher-level system,
 that system may have it's own way of create a `PangoContext`.
 For instance, the GTK toolkit has, among others,
 `gtk_widget_get_pango_context()`. Use those instead.</doc>
-        <return-value transfer-ownership="full">
```

Comment:
@GuillaumeGomez Here are your fixes

Reply:
Yeay!

Reply:
Unfortunately merging this is blocked by a GTK4 fix now :)

Maybe we should have separate CI jobs for each of the projects?